scopes/react/ui/properties/properties.tsx

```diff
@@ -1,9 +1,21 @@
 import { LinkedHeading } from '@teambit/documenter.ui.linked-heading';
 import { PropTable } from '@teambit/documenter.ui.property-table';
 import { Section } from '@teambit/documenter.ui.section';
+import { useFetchDocs } from '@teambit/component.ui.hooks.use-fetch-docs';
 import React from 'react';
 
-export function Properties({ properties }: any) {
+export type PropertiesParams = {
+  componentId: string;
+}
+
+export function Properties({ componentId }: PropertiesParams) {
+  const { loading, error, data } = useFetchDocs(componentId);
```

Comment:
moving the fetch here could potentially really improve loading times.

however, what if we want to use docs data in other places?  (ie in CompositionsSummery, etc)

Reply:
And i guess that's the point of the hook, you can use it wherever you need to (and it's memo-ised so shouldnt be a loading issue)

Reply:
I guess. Good.